terraform/aws/modules/cluster-post-installation/grafana_dashboard_configmaps.tf

```
@@ -139,3 +137,31 @@ resource "kubernetes_config_map" "mattermost_performance_monitoring_per_cluster"
     "mattermost_performance_monitoring_per_cluster.json" = "${file("../../../../grafana-dashboards/mattermost_performance_monitoring_per_cluster.json")}"
   }
 }
+
+resource "kubernetes_config_map" "skydns" {
+  metadata {
+    name      = "skydns-dashboard"
+    namespace = "monitoring"
+    labels = {
+      cloud_dashboards = "skydns"
+    }
+  }
+
+  data = {
+    "coredns.json" = "${file("../../../../grafana-dashboards/skydns.json")}"
+  }
+}
+
+resource "kubernetes_config_map" "uptime" {
```

I would suggest to follow the pattern
```suggestion
resource "kubernetes_config_map" "uptime_view" {
```

---

@angeloskyratzakos Agree. Need to do follow up changes on other places too, if we all agree  on this convention.

---

not talking about naming convention in general, but on the specific resource, lines +157 to +165